src/vm/eventpipesession.cpp

```diff
@@ -365,6 +365,7 @@ CLREvent *EventPipeSession::GetWaitEvent()
     return m_pBufferManager->GetWaitEvent();
 }
 
+// MUST be called AFTER sending the IPC response
 void EventPipeSession::Enable()
```

Comment:
I recommend we rename this, how about EventPipeSession::StartStreaming()?

The current name strongly implies that it is part of the EventPipe::Enable() operation and that is no longer the case.

Reply:
I think I had done that with that exact name in a different draft of these changes.  I'll make the update and comment accordingly 👍